galaxy/values.yaml

```diff
@@ -10,7 +10,7 @@ image:
   #- Repository containing the Galaxy image.
   repository: quay.io/galaxyproject/galaxy-min
   #- Galaxy Docker image tag (generally corresponds to the desired Galaxy version)
-  tag: "25.0.2"  # Galaxy versions prior to 24.1.1 contain a bug mapping the extra_files directory
+  tag: "25.1-auto"  # Galaxy versions prior to 24.1.1 contain a bug mapping the extra_files directory
```

Comment:
-1 on having the default image being the auto build.

Reply:
Given that we now have patch releases for Galaxy - it would make sense to pin to an exact version. The major downside is that we have to continuously perform those releases. Can we automate perhaps?